docs/commands.md

```diff
@@ -0,0 +1,87 @@
+---
+id: commands
+title: "Commands"
+---
+
+`Command[Model]` is a description of a CLI command. This is parsed by `CliApp` to produce a collection of valid commands and transform a valid input from a user into an instance of type `Model`. Then you can implement the functionality of the CLI App using pattern matching on instances of `Model`.
+
+
+
+It's possible to create a `Command` using the `apply` method. You must always specify the name of the command that will be used
+in the CLI, but you can also specify options and/or arguments:
+
+```scala mdoc:silent
+import zio.cli._
+
+object Command {
+  def apply(name: String) = ???
+  def apply[ArgsType](name: String, args: Args[ArgsType]) = ???
+  def apply[OptionsType](name: String, options: Options[OptionsType]) = ???
+  def apply[OptionsType, ArgsType](name: String, options: Options[OptionsType], args: Args[ArgsType]) = ???
+}
+```
+## Combining commands and transformations
+**ZIO CLI** also allows to combine different commands in a functional manner.
```

Comment:
As with the other section, consider breaking these into families and documenting each one separately, rather than just listing the trait. Adopt a use-case-oriented way of explaining them. For example: we want our program to have two commands, how do we do that?  We want our program to have a subcommand, how do we do that? (Of course, you have to define and explain the utility of subcommands.)